examples/windows-plugin/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Sonobuoy contributors
+# Copyright 2019 Sonobuoy contributors
```

Comment:
Should yr reflect current yr ?

Reply:
No, it should reflect the year in which it was written and apparently it was 2019 in the original so as we copy it over that should stay. Thats my understanding.